test/powershell/Language/LanguageTestSupport.psm1

```diff
@@ -76,7 +115,7 @@ function ShouldBeParseError
         if ($SkipAndCheckRuntimeError) 
         {
             It "error should happen at parse time, not at runtime" -Skip {}
-            $errors = Get-RuntimeError -Src $src
+            $errors = Get-RuntimeError -Src $src -Timeout 5

```

Comment:
it was SWAG for sure, I don't have trouble with having this be 10

---
In reply to: [94878763](https://github.com/PowerShell/PowerShell/pull/2958#discussion_r94878763) [](ancestors = 94878763)